src/Server/Common/CustomResourceWatcher.cs

```diff
@@ -84,8 +84,8 @@ public void Start(double interval)
 
         public void Stop()
         {
-            _timer.Stop();
-            _timer.Dispose();
+            _timer?.Stop();
+            _timer?.Dispose();
             _logger.Log(LogLevel.Information, $"{GetType()} watcher stopped.");
```

Comment:
you more than likely mean `$"{GetType().Name} watcher stopped."`, or you can use `.FullName` if that's a better fit.